src/babel.js

```diff
@@ -54,8 +57,17 @@ module.exports = ({ types: t }) => {
           t.JSXAttribute(t.JSXIdentifier(propName))
         )
 
+        const propMeta = !prop || prop === null ? 'null' : hash(JSON.stringify(prop))
+        const hashId = hash(`${filename}_${name}_${propMeta}`).toString(16)
+
+        if (attributeMap[hashId]) {
```

Comment:
Otherwise we will get:
![image](https://user-images.githubusercontent.com/16756393/39128159-bd2adeee-470f-11e8-977a-302e361a8763.png)